### PR TITLE
Add stats to stdout storage

### PR DESCRIPTION
**Pull request by piowag**
Add stats to stdout storage: memory, hugetlb, perf, resctrl, referenced memory

### CODE REVIEW

File: cmd/internal/storage/stdout/stdout.go [hunk 1]
```diff
@@ -32,27 +33,62 @@ type stdoutStorage struct {
 }
 
 const (
-	colTimestamp = "timestamp"
-	// CPU Uasge
-	colCpuCumulativeUsage = "cpu_cumulative_usage"
```
Comment (dashpole) [inline cmd/internal/storage/stdout/stdout.go:37]: I'm fine with the rename, but just in-case someone is using these metrics, lets keep the old name around as well, with a link to an issue to remove it in a few releases (v0.39.0)

**Comment by Creatone** [comment]
cc: @Creatone

**Comment by dashpole** [comment]
This looks good.  Do you think we have enough metrics here where we need to start respecting --ignore-metrics?  I believe the current PR will just have zero values for disabled metrics, which is acceptable, but adds to the load for any backends that process the data.

**Review by dashpole — APPROVED** [review]
lgtm

**Pull request merged**